tests/integration/Forms/FormCest.php

```diff
@@ -14,6 +14,7 @@
 use IntegrationTester;
 use Phalcon\Forms\Element\Text;
 use Phalcon\Forms\Form;
+use Phalcon\Html\Interfaces\AttributesInterface;
```

Comment:
In the `tests/_ci/` there is a file called `generate-test-stubs.php`. 

Run that file. After it is done inside the `tests` folder you will have a `nikos/Phalcon` folder with all the namespaces. Each namespace will contain one Cest per method. You can use those now and copy them in the suite accordingly and split this big test into smaller ones that makes it easier for everyone to understand ;)

Reply:
Following the collection object you can adjust the code accordingly. The benefit is reusing the component and also take advantage of its logic i.e. keep things DRY. Also case insensitivity is available.